Entities/Attachable.cpp

```diff
@@ -315,31 +353,6 @@ bool Attachable::TransferJointForces(Vector &jointForces)
          fItr != m_Forces.end(); ++fItr)
         forces += fItr->first;
 
-/* doesn't really apply to forces, only impulse forces
-    // Factor in the stiffness, or lack thereof
-    forces *= m_JointStiffness;
```

Comment:
Because it says "doesn't really apply to forces, only impulse forces" and is commented out.
Exactly the same code is present and active in Attachable::TransferJointImpulses.
It's just clutter.

Reply:
What suggests it only handles impulse forces? It iterates through m_Forces, which I believe is non-impulse forces on line 314/352.

It seems like with this if you apply force through lua it won't cause breakwounds and stuff, cause you got rid of that. That said, it's a lot of duplicated code that I think should be extracted into one method that's called for impulse forces and regular forces, but that's a separate problem.

Reply:
Fuuuuck. Sorry, I'm blind, didn't see that the /* wasn't a single line thing. Nevermind all of this lol